src/index.js

```diff
@@ -73,7 +73,7 @@ class HMIApp extends React.Component {
         this.onTouchMove = this.onTouchMove.bind(this);
         this.onTouchEnd = this.onTouchEnd.bind(this);
         this.onTouchEvent = this.onTouchEvent.bind(this);
-        store.dispatch(resetTimeout(props.resetPeriodValue));
+        store.dispatch(resetTimeout({resetPeriod: 10000}));  
```

Comment:
Do not use magic numbers. Let's use `DEFAULT_RESET_TIMEOUT` from src/js/Alert.js instead

Reply:
@YuriiYefimenko Alert.js is a component. This is not a file with constants value. It's wrong to use like you say

Reply:
@ValeriiMalkov, @YuriiYefimenko, I will create  DEFAULT_RESET_TIMEOUT const for index file. Ok?

Reply:
@ValeriiMalkov, @YuriiYefimenko, fixed.